actors/evm/tests/contracts/NotificationReceiver.sol

```diff
@@ -0,0 +1,622 @@
+// SPDX-License-Identifier: MIT
+pragma solidity 0.8.25;
+
```

Comment:
```suggestion

// ========================================================
// NOTE: If using this contract as an example, consider depending on utilities
// available in https://github.com/filecoin-project/filecoin-solidity instead of
// copying reusable utilities from here.
// ========================================================
```